src/devices/Is31fl3730/Is31fl3730.cs

```diff
@@ -0,0 +1,381 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Device.I2c;
+using System.Threading;
+using static System.Linq.Enumerable;
+
+namespace Iot.Device.Display
+{
+    /// <summary>
+    /// Represents an IS31FL3731 LED Matrix driver
+    /// </summary>
+    // Datasheet: https://cdn-shop.adafruit.com/product-files/3017/31FL3730.pdf
+    // Product: https://shop.pimoroni.com/products/microdot-phat
+    // Product: https://shop.pimoroni.com/products/led-dot-matrix-breakout
+    // Related repo: https://github.com/pimoroni/microdot-phat
+    // Based on: https://github.com/pimoroni/microdot-phat/blob/master/library/microdotphat/matrix.py
+    public class Is31fl3730 : IDisposable
+    {
+        // Function register
+        // table 2 in datasheet
+        private const byte CONFIGURATION_REGISTER = 0x0;
```

Comment:
rather than using constants, use a Register.cs file with enums in there being internal. Like this, you will be able to keep the register names like they are (all uppercase). And it's much better than constants.